plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -77,6 +78,16 @@ class Bazel(
 
     override fun command(workingDir: File?) = "bazel"
 
+    override fun run(vararg args: CharSequence, workingDir: File?, environment: Map<String, String>): ProcessCapture =
+        super.run(
+            args = args,
+            workingDir = workingDir,
+            environment = environment.toMutableMap().apply {
+                // Disable the optional wrapper script under `tools/bazel`, to ensure the --version option works.
+                put("BAZELISK_SKIP_WRAPPER", "true")
+            }
```

Comment:
I propose to use the shorter `environment + mapOf("BAZELISK_SKIP_WRAPPER" to "true")`.